tests/test_db_function.py

```diff
@@ -305,8 +305,8 @@ def test_get_progress_for_application():
     )
 
     assert len(application_progress) == 2
-    assert application_progress[0]["progress"] == 22
-    assert application_progress[1]["progress"] == 11
+    assert application_progress[0]["progress"] == 20
+    assert application_progress[1]["progress"] == 10
```

Comment:
These also change due to config, so one might argue this test could also be disabled, I didn't write this one though, so have just fixed it for now.